app/controllers/service_types_controller.rb

```diff
@@ -3,8 +3,40 @@ class ServiceTypesController < ApplicationController
 
   # Authentication by default inherited from ApplicationController.
 
+  before_action :authorize_for_districtwide_access_admin, except: [:index]
+
+  def authorize_for_districtwide_access_admin
+    unless current_educator.admin? && current_educator.districtwide_access?
```

Comment:
Should this be `can_set_districtwide_access?`  This will give access to a more people (eg, `test_pals.rich` as an example).